bug_test_files

PeijinZhou opened the pull request:
Each file could have many bugs(At least have a bug as the name)

PeijinZhou committed:
bug_test_files

Each file could have many bugs(At least have a bug as the name)

IncapableFury commented:
Can you specifically indicate the errors within each file(in the NOTE field for example)? Otherwise, the developer can't check if his functions are correct.

PeijinZhou commented:
> Can you specifically indicate the errors within each file(in the NOTE field for example)? Otherwise, the developer can't check if his functions are correct.

Of course, I will push a file without mistake, all the bug test file is fixed by that one.

PeijinZhou committed:
No_bug_file

This file is used for testing whether the user stories could run successfully.  All the test flies is fixed by this file. You can also use the file in testing_files for test.

PeijinZhou committed:
bug_test_file

I have marked all mistakes in each file.

IncapableFury commented:
Can't merge until the testing strategy is determined
